packages/runtime/test/src/logicsig.ts

```diff
@@ -1,27 +1,66 @@
 import { decodeAddress, multisigAddress, MultisigMetadata } from "algosdk";
 import { assert } from "chai";
-
+const { convert } = require("@algo-builder/algob");
```

Comment:
I think you can use `import` instead of const

Reply:
Done